### PR TITLE
Added Responsiveness to Home Page

### LINKED ISSUE #127: Overlap while loading events
check this image
Screenshot from 2020-09-30 16-53-49

### LINKED ISSUE #134: Event and Partner Section on Homepage
### Description
There is a lot of blank space being generated beside the event section and the containers of about and partner section appear misaligned w.r.t the community section for large resolution screen (mainly width above 1920 px).

### Screenshots

Screenshot (176)
Screenshot (175)

### PR DESCRIPTION
**Pull request by hariom1625**
fixes #127 
fixes #134 

Description:  
* Fixed occurrence of blank space in the about section
* Fixed overlapping of the union while loading of events 
* Added responsiveness to the home page for large resolution screen.

#### Checklist
- [x] I've followed the Contributing guidelines provided in the repository.
- [x] I've made the changes that were demanded in the linked issue.
- [x] I've tested my code on a Chromium-based browser.
- [x] I've tested my code on Mozilla Firefox.
- [x] My code gave a clean console on debugging. (no warnings/errors)

### CODE REVIEW

**Comment by ankiiitraj** [comment]
Is removing the planet from smaller screens intended? @hariom1625 
The rest Looks good to me! 🎉

**Comment by hariom1625** [comment]
> Was removing the planet from smaller screens was intended? @hariom1625
> The rest Looks good to me! 🎉

That was intended but I think I screwed up something!
Thanks for pointing that out :-)

**Comment by EmperorYP7** [comment]
image

Still looks a bit messy.. 🤔

File: client/src/components/Community/Community.css [hunk 1]
```diff
@@ -10,7 +10,8 @@
 }
 
 .community-component .union-img {
-  height: 250vh;
+  margin-top: 20vh;
```
Comment (EmperorYP7) [inline client/src/components/Community/Community.css]: I think its not lining up with the corresponding title. This looks so unsatisfying to me..

**Comment by hariom1625** [comment]
> image
> 
> Still looks a bit messy.. 🤔

I have added media queries only for the large resolution screens as mentioned in the issue #134 .
I think a new issue can be opened for optimizing it for smaller resolutions.
Or I can fix this in my next commit :-)

File: client/src/components/Community/Community.css (hunk 1 above)
Comment (hariom1625) [reply client/src/components/Community/Community.css]: Not clear, Can you ellaborate further !!

Comment (EmperorYP7) [reply client/src/components/Community/Community.css]: The circle of `union.svg` is not lining up with "From the Geeks' Satellite.." which is what the design aims for. I think achieving that alignment is more fundamental aspect of the design.

**Comment by ankiiitraj** [comment]
> > image
> > Still looks a bit messy.. thinking
> 
> I have added media queries only for the large resolution screens as mentioned in the issue #134 .
> I think a new issue can be opened for optimizing it for smaller resolutions.
> Or I can fix this in my next commit :-)

@EmperorYP7 Your call!

**Comment by EmperorYP7** [comment]
Well, it would be great if you solve this here itself @hariom1625.

File: client/src/components/Community/Community.css (hunk 1 above)
Comment (hariom1625) [reply client/src/components/Community/Community.css]: Okay, Got it :-)

**Comment by hariom1625** [comment]
> > > image
> > > Still looks a bit messy.. thinking
> > 
> > 
> > I have added media queries only for the large resolution screens as mentioned in the issue #134 .
> > I think a new issue can be opened for optimizing it for smaller resolutions.
> > Or I can fix this in my next commit :-)

For displaying the events for resolutions varying between desktop and mobile,I think we will need to change the layout because if we will use this same layout it appears a lot messy as the left place is left all blank and on the right all of the events are stacked and we can't shift that vertical line as it is required for the line to go from between because of the community section present below event section.
Suggestions ?

**Comment by EmperorYP7** [comment]
> For displaying the events for resolutions varying between desktop and mobile,I think we will need to change the layout because if we will use this same layout it appears a lot messy as the left place is left all blank and on the right all of the events are stacked and we can't shift that vertical line as it is required for the line to go from between because of the community section present below event section.
Suggestions ?

For this, we can modify the media queries to change the layout at a screensize where things start to look messy.

**Comment by hariom1625** [comment]
Have a look once again I have changed the layout a bit for (resolution <750px).

**Comment by EmperorYP7** [comment]
This is much better :+1:

**Comment by ankiiitraj** [comment]
squash the commits and push! and this is good to go

**Review by ankiiitraj — APPROVED** [review]

**Comment by ankiiitraj** [comment]
while pulling use `git pull upstream front-end --rebase`, this will not generate a merge commit

**Review by EmperorYP7 — APPROVED** [review]

**Pull request merged**